python/paddle/v2/fluid/tests/test_one_hot_op.py

```diff
@@ -0,0 +1,46 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+import math
+from op_test import OpTest
+import paddle.v2.fluid.core as core
+
+
+class TestOneHotOp(OpTest):
```

Comment:
It will also help to have some negative examples, where the depth is less than the index values. We can catch the exception thrown by the executor to make sure that the code gives an error for such faulty cases.

Reply:
Thanks, have added more unit test.